ios/src/main/java/java/time/LocalDate.java

```diff
@@ -0,0 +1,12 @@
+package java.time;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Hack file to make AppKit work on JRE7
+ */
+public class LocalDate {
+    public static LocalDate parse(String date, DateTimeFormatter formatter) {
+        throw new UnsupportedOperationException();
+    }
+}
```

Comment:
Shouldn't be moved as it would clash when code is launched on JRE8. If I remember correctly, LocalDate was used (imported) in some Explorer API wrappers but not actually used in the needed APIs, the Exception is not raised. So just providing this as a workaround file is okay for now in my opinion until we actually need the code.